tests/test_util.py

```diff
@@ -52,6 +53,93 @@ def test_update_dict_by_adding_another_adds_shared_arrays_reversed():
     assert len(dict2.keys()) == 2
 
 
+class DummyPrognostic(Prognostic):
+    input_properties = {'temperature': {'alias': 'T'}}
+    diagnostic_properties = {'pressure': {'alias': 'P'}}
+    tendency_properties = {'temperature': {}}
+
+    def __init__(self):
+        self._a = 1
+
+    def __call__(self, state):
+        return self._a
+
+
+def test_get_component_aliases_with_different_input_types():
```

Comment:
You should split this into three separate tests :) That way even if the first test fails, the last two tests still run.